hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/TransactionProcessor.java

```diff
@@ -183,8 +178,8 @@ private HederaEvmTransactionResult commitResourceExhaustion(
      * </ol>
      *
      * <p>Note that if the transaction is a {@code CONTRACT_CREATION}, setup includes calling either
-     * {@link HederaWorldUpdater#setupCreate(Address)} or
-     * {@link HederaWorldUpdater#setupAliasedCreate(Address, Address)}.
+     * {@link HederaWorldUpdater#setupInternalCreate(Address)} or
```

Comment:
Are these supposed to be `setupTopLevelCreate` and `setupAliasedTopLevelCreate`?

Reply:
Good catch, fixed!